app/src/main/java/com/ncnf/utilities/map/MapHandler.java

```diff
@@ -140,21 +143,21 @@ private void addEventMarkers(List<Event> events){
                 desc.append(p.getName()).append("\n");
             }
             String description = desc.toString();
-            clusterManager.addItem(new CustomMapMarker(k, description, eventMap.get(k).get(0).getAddress(), list, true));
+
+            clusterManager.addItem(new CustomMapMarker(k, description, eventMap.get(k).get(0).getAddress(), list, null, true));
```

Comment:
Don't pass a null reference as a list attribute. If you really want to have a "null" instead of an empty list, use Optional.empty

Reply:
This is not for the list, this is for the Organization where I saw there's no "blank" constructor. I agree this might not be the cleanest but it seemed the simplest to me

Reply:
This is not for the list, this is for the Organization where I saw there's no "blank" constructor. I agree this might not be the cleanest but it seemed the simplest to me